VideoLocker/res/layout/fragment_user_profile.xml

```diff
@@ -0,0 +1,181 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:orientation="vertical">
+
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:background="@color/edx_brand_primary_base"
+            android:gravity="center"
+            android:orientation="vertical"
+            android:padding="@dimen/edx_margin">
+
+            <de.hdodenhof.circleimageview.CircleImageView
+                android:id="@+id/profile_image"
+                style="@style/profile_image"
+                android:layout_width="166dp"
+                android:layout_height="166dp"
+                app:civ_border_width="3dp"
```

Comment:
Again, maybe we should have these dimensions moved to a global `dimen` resource?